collections/_article/godot-foundation-update-sep-2024.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Godot Foundation Update September 2024"
+excerpt: "We've been quietly working on a number of things! Here are a few."
+categories: ["news"]
+author: Godot Foundation
+image: /storage/blog/covers/godot-foundation-update-2023.jpg
+date: 2024-09-30 12:00:00
+---
+
+The Foundation has been working on a number of things behind the scenes that are important for the organization and for
+the Godot ecosystem more broadly. We wanted to take the chance to highlight a few of them in this post.
+
+## Key Policies and Procedures
+The Foundation was formed approximately two years ago, but we have only been operating in a full capacity since last
+summer. We initially adopted the [member project policies](https://sfconservancy.org/projects/policies/) of the Software
```

Comment:
Probably best to be more explicit than "last summer". Does that mean summer 2023 or summer 2024? And do you mean summer in northern hemisphere?